Cargo.toml

```diff
@@ -32,6 +32,7 @@ clap = { version = "^4.1.6", features = ["derive", "env"] }
 color-eyre = "^0.6.2"
 ctrlc = { version = "^3.2.5", features = ["termination"] }
 either = "^1.8.1"
+glob = "^0.3.1"
```

Comment:
This version of [glob](https://crates.io/crates/glob) (the latest at the time of writing) kind of sucks because it pattern matches with UTF-8 strings rather than path strings. There may be edge cases in there, esp. if I try to port this to work on Windows.